app/mailers/rents_mailer.rb

```diff
@@ -6,6 +6,6 @@ class RentsMailer < ApplicationMailer
   #
   def new_rent_notification(rent_id)
     @rent = Rent.find(rent_id)
-    mail to: @rent.user.email, from: 'no-reply@wolox.cl', subject: 'You just rented a book'
+    mail to: @rent.user.email, from: 'no-reply@wolox.cl'
```

Comment:
you forgot to put the translation of the subject here?

Reply:
If is not set here the mailer tries to find it in the translations files... fetching for current locale or default locale otherwise